src/adam_assist/__init__.py

```diff
@@ -92,15 +105,230 @@ def hash_orbit_ids_to_uint32(
     
     return mapping, hashes
 
+@ray.remote
+def assist_propagation_worker_ray(
+    orbits: OrbitType,
+    times: OrbitType,
+    adaptive_mode: int,
+    min_dt: float,
+    propagator: Type["Propagator"],
+    **kwargs,
+) -> OrbitType:
+    prop = propagator(**kwargs)
+    propagated = prop._propagate_orbits(orbits, times, adaptive_mode, min_dt)
+    return propagated
+
 
 class ASSISTPropagator(Propagator):
     def _propagate_orbits(
-        self, orbits: OrbitType, times: TimestampType
+        self, orbits: OrbitType, times: TimestampType, adaptive_mode: int, min_dt: float
```

Comment:
`min_dt` needs to either be nullable the whole way through or we need to guarantee we pass a value down to it if `None` in the higher level function all.

Reply:
Looks like still required here?
https://github.com/B612-Asteroid-Institute/adam-assist/pull/3/files#diff-b351543f18e0080a3b77556f79319993eda742115cf9081396cd919df670b3d3R113

Reply:
Okay, should be fixed now. Sorry, not sure what happened there.